plugins/services/src/js/components/MarathonTaskDetailsList.js

```diff
@@ -82,7 +82,7 @@ class MarathonTaskDetailsList extends React.Component {
       };
 
       return (
-        <DescriptionList
+        <HashMapDisplay
```

Comment:
I agree! I actually started doing this as part of this PR, but eventually decided that it should be done in a different task so as to keep this PR smaller. I just created a JIRA so we can track this: https://mesosphere.atlassian.net/browse/DCOS-12416